.github/workflows/ci.yml

```diff
@@ -325,6 +325,19 @@ jobs:
           body_path: ${{ github.workspace }}-CHANGELOG.txt
           draft: false
 
+  update-homebrew-formula:
+    needs:
+      - go-release-cli
+    if: github.event_name == 'push' && startsWith(github.ref, 'refs/tags/releases/mcap-cli/v')
+    runs-on: ubuntu-latest
+    steps:
+      - uses: mislav/bump-homebrew-formula-action@v2
```

Comment:
Standard supply chain risk, not dissimilar from zillions of npm packages we import. The PRs this action creates still have to be manually approved by homebrew maintaineres.

If we're concerned about it, pinning the shasum (rather than major version branch) is probably simpler than forking it.

Reply:
I pinned the sha